File: backend/app/models/manager.py
# SQLAlchemy ORM model: managers table
# Columns (from CSV):
#   id (UUID PK), session_id (str, indexed), full_name, position (Specialist/SeniorSpecialist/ChiefSpecialist),
#   skills (ARRAY of str: VIP/ENG/KZ), business_unit (str → FK or name),
#   workload (Int, number of active tickets — incremented on each assignment)

File: frontend/src/types/manager.ts
export type Position = 'Specialist' | 'SeniorSpecialist' | 'ChiefSpecialist'

export type Skill = 'VIP' | 'ENG' | 'KZ'

export interface Manager {
  id: string
  fullName: string
  position: Position
  skills: Skill[]
  businessUnit: string
  workload: number
  sessionId: string
}
